test/onnx/gen_onnx.py

```diff
@@ -777,6 +777,25 @@ def deconv_stride_test():
     return ([node], [x, w], [y])
 
 
+@onnx_test
+def div_const_test():
+    x = helper.make_tensor_value_info('0', TensorProto.FLOAT, [3])
+    y = np.array([1.0, 2.0, 3.0])
+    y_tensor = helper.make_tensor(name='1',
+                                  data_type=TensorProto.FLOAT,
+                                  dims=y.shape,
+                                  vals=y.flatten().astype(np.float))
+
+    z = helper.make_tensor_value_info('2', TensorProto.FLOAT, [3])
+
+    node = onnx.helper.make_node('Div', inputs=[
+        '0',
+        '1',
+    ], outputs=['2'])
+
+    return ([node], [x], [z], [y_tensor])
```

Comment:
Is this necessary? There is no onnx test for it. Also, I dont think we need an onnx test.

Reply:
fixed